### PR TITLE
feat: [sc-25927] Update ENS Referral Program Landing Page for DevCon

### PR DESCRIPTION
**Pull request by eduramme**
Story details: https://app.shortcut.com/ps-web3/story/25927

### CODE REVIEW

File: apps/namehashlabs.org/app/ens-referral-program/page.tsx [hunk 1]
```diff
@@ -44,8 +44,10 @@ export default function Page() {
                   alt="hero"
                 />
               }
-              greenLabelText="Proposal pending"
-              buttonUrl="https://docs.google.com/document/d/1srqcho7PFyMBUDQTxxlH_eZqrt5x_EEB-PF2LfpYvIg/edit?usp=sharing"
+              calendarButtonText="Meet us at DevCon"
+              greenLabelText="Collecting Community Input"
+              buttonText="Share your voice"
+              buttonUrl="https://www.deform.cc/"
```
Comment (eduramme) [inline apps/namehashlabs.org/app/ens-referral-program/page.tsx:50]: @lightwalker-eth just a heads-up that I added the link https://www.deform.cc/ here

File: apps/namehashlabs.org/data/ensProfiles.ts [hunk 2]
```diff
@@ -467,7 +467,7 @@ const EnsProfiles = new Map<`0x${string}`, Profile>([
   [
     "0x89F8e4020c0dd384F13c288bc5743F963F9D8fdF",
     buildProfile({
-      ensName: "edulennert.eth",
+      ensName: "duds.eth",
```
Comment (lightwalker-eth) [inline apps/namehashlabs.org/data/ensProfiles.ts:470]: @eduramme Could you please set the avatar on your ENS profile?

CleanShot 2024-11-09 at 21 09 21

I see you never set any avatar:

CleanShot 2024-11-09 at 21 10 20

File: apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx [hunk 3]
```diff
@@ -1,24 +1,25 @@
...
 
 export const HeadlineBanner = () => {
   return (
     <div className="px-5 text-white justify-center items-center w-full bg-[#3B0D48] space-x-4 py-4 text-sm relative z-20 flex">
       <div className="flex space-x-3 items-center">
         <MegaphoneIcon className="flex shrink-0 w-6 h-6 opacity-50 -mr-1" />
-        <p className="pr-16 sm:pr-0">
-          Track our progress as an ENS Service Provider
-        </p>
+        <p className=" sm:pr-0">Meet us at DevCon in Bangkok 12-15 Nov 2024</p>
```
Comment (lightwalker-eth) [inline apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx:10]:
```suggestion
        <p className=" sm:pr-0">Meet us at DevCon in Bangkok 11-13 Nov 2024</p>
```

We'll be there only 11-13 :+1:

File: apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx [hunk 4]
```diff
@@ -1,24 +1,25 @@
 import { MegaphoneIcon } from "@heroicons/react/24/outline";
+import { CalendarButton } from "@namehash/internal";
 import { Link } from "@namehash/namekit-react";
 
 export const HeadlineBanner = () => {
```
Comment (lightwalker-eth) [inline apps/namehashlabs.org/components/1 - atoms/headline-banner.tsx:5]: Please fix the height of the banner. It's too big.

CleanShot 2024-11-09 at 21 13 42

**Review by lightwalker-eth — CHANGES_REQUESTED** [review]
@eduramme Hey thanks for the updates here. There's a few items to fix. Appreciate if you can fix these today. Thanks :+1:

**Review by lightwalker-eth — APPROVED** [review]
@eduramme Hey looks great, thank you :+1:

**Pull request merged**